src/static/css/_globals.scss

```
@@ -307,6 +307,15 @@ nav#main-navbar {
     ul.navbar-menu {
         @extend .navbar-nav;
 
+        & > li:not(.user-menu):nth-child(4) > a {
+            font-weight: bolder;
+            border: 3px solid #DB1C83;
```

Même si je suis coupable de ça à l'occasion, il faudrait éviter d'écrire les couleurs en dur dans le css, car nous avons un fichier `theme.scss` qui définit les variables de couleur.

```suggestion
            border: 3px solid $contrast-color;
```